prisma/schema.prisma

```diff
@@ -211,3 +211,21 @@ model UserProposalFeedback {
 
   @@unique([proposalId, userEmail])
 }
+
+model AdminInfo {
```

Comment:
maybe we should build it in the app backend such that the admin info is prepopulated when the proposal is created -> the title and other related info could be pre-filled and could be made required, which makes more sense than having it nullable